.github/workflows/irc-notifications.yaml

```diff
@@ -107,6 +107,41 @@ jobs:
             ${{ env.SUMUP }}\n\
             ${{ github.event.compare }}"
 
+      - name: Shorten PR fields
+        if: github.event_name == 'pull_request'
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
+          BODY:  ${{ github.event.pull_request.body }}
+        run: |
+          # -------------------------------------
+          echo "# script parse.pl"
+          cat <<'EOS' > parse.pl
+          use v5.14; use strict; use warnings;
+          my ( $key, $len ) = @ARGV;
+          $key //= ''; $len //= 50;
+          my $txt = substr( $ENV{$key} || "Unknown $key", 0, $len );
+          $txt =~ s{\\n}{\n}g; $txt =~ s{\\t}{ }g; $txt =~ s{\t}{ }g;
+          $txt =~ s{^EOS}{}mg;
+          say $txt;
+          EOS
+
+          # -------------------------------------
+          echo 'TITLE<<EOF'       >> $GITHUB_ENV
```

Comment:
Seems like this should be using `::set-output` rather than the environment.

Reply:
note: the `$GITHUB_ENV` is still documented and works fine
any reasons to stop using it?
view https://docs.github.com/en/enterprise-server@3.4/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Outputs just seem like the more appropriate way to communicate between these actions. And I was wrong about `::set-output`, as it is deprecated. `$GITHUB_OUTPUT` would be the appropriate thing to use for outputs. But none of this is a big deal. If it's more annoying to use outputs, it doesn't need to be changed.

Reply:
I understand, not against using `$GITHUB_OUTPUT`. I'm not sure how to consume it later in other workflow without some research. Will have a look.

Reply:
giving it a try using 79728f9c044f61881c5c6eb25404c1245967512c 
this is going to require a longer syntax where it's used but should be fine and cleaner, thanks for the suggestion
`$steps.sumup.output.SUMUP` instead of `env.SUMUP`

Reply:
@haarg after multiple attempts I was able to find the proper way to exchange variables between steps & jobs.
But I cannot make it work using multilines variables like the one currently used by sumup.
I'm going to keep using `GITHUB_ENV` for now, patch welcome.
thanks

Reply:
here is probably the solution to use multiline variables for an output variable...
https://trstringer.com/github-actions-multiline-strings/
honestly the current usage of ENV make it much easier to maintain at this point

Reply:
feel free to reopen the discussion if needed